lib/core.js

```diff
@@ -222,6 +222,14 @@ function Core(options)
 
           GlobalConfig.setKeepAlive(keepAlive);
         }
+
+        let customCredentialManager = options.customCredentialManager;
```

Comment:
i assume this is to cover a scenario where user sets it via connectionConfig(options) instead of GlobalConfig?

Reply:
Do you mean this option needs to go to the connection config?

Reply:
Sorry, my bad - misunderstood this code. We don't want to do it via connection config.